spring-native/src/main/java/io/opentelemetry/example/graal/CustomHints.java

```diff
@@ -0,0 +1,33 @@
+package io.opentelemetry.example.graal;
+
+import io.opentelemetry.exporter.otlp.logs.OtlpGrpcLogRecordExporterBuilder;
+import io.opentelemetry.exporter.otlp.metrics.OtlpGrpcMetricExporterBuilder;
+import io.opentelemetry.exporter.otlp.trace.OtlpGrpcSpanExporterBuilder;
+import org.springframework.aot.hint.MemberCategory;
+import org.springframework.aot.hint.RuntimeHints;
+import org.springframework.aot.hint.RuntimeHintsRegistrar;
+import org.springframework.aot.hint.TypeReference;
+
+public class CustomHints implements RuntimeHintsRegistrar {
+
+  @Override
+  public void registerHints(RuntimeHints hints, ClassLoader classLoader) {
+
+    hints
+        .reflection()
+        .registerType(
+            TypeReference.of(
+                "io.opentelemetry.instrumentation.spring.autoconfigure.aspects.InstrumentationWithSpanAspect"),
+            MemberCategory.INVOKE_PUBLIC_METHODS,
+            MemberCategory.INVOKE_DECLARED_METHODS);
+    hints
```

Comment:
If the following configurations are required with the new example, it would be great to add them in the [opentelemetry-java-instrumentation](https://github.com/open-telemetry/opentelemetry-java-instrumentation) repository.